docs/MONITORING.md

```diff
@@ -0,0 +1,123 @@
+
+### 1. Getting your APM Server URL <a name="server-url"></a>
+Navigate to your Kibana URL and select the space that you would like to associate with your frontend app (default if you don't have several spaces in Kibana). Then open the drawer menu and navigate to `APM`:
+
+![Navigation instructions to APM in Kibana](/docs/public/kibana-screenshot-1.png)
+
+In the top right corner of this page click on `Add data`:
+
+![Navigation instructions to the Add data button in APM](/docs/public/kibana-screenshot-2.png)
+
+Then scroll down to APM Agents and select `RUM (JS)`:
+
+![Navigation instructions to the RUM Agent](/docs/public/kibana-screenshot-3.png)
+
+There are two suggested code blocks for setting up the RUM Agent, find in any of them the `serverUrl` param and copy/paste it somewhere in your notes, we will use this URL later in a posterior step.
+
+*Note: You have two options for getting your init script, you either install the `@elastic/apm-rum` dependency in your project or you set up the agent with `<script>` tags. In this document we will describe an approach for the latter.*
+
+### 2. Using the /public/init-js script <a name="init-js"></a>
+
+The __compas-open-scd__ project features a reference to an "empty" javascript resource at `index.html` (line:42)
+```html
+  <script src="./public/init-js/init.js"></script>
+```
+
+This init javascript file has the purpose to allow for dynamic configuration of each compas-open-scd deployment. 
+
+In a private repository create a copy of `init.js` (or edit if you already have your own copy of the file with custom init scripts) in a folder, let's call it `path-to-init-js` and add the following code:
+
+```js
+const script = document.createElement('script');
+script.type = 'text/javascript';
+// download and host your preffered RUM Agent version minified from https://github.com/elastic/apm-agent-rum-js/releases
+script.src = 'https://your-cdn-host.com/path/to/elastic-apm-rum.umd.min.js'; 
+script.async = true;
+script.crossorigin = "anonymous"; // provides support for CORS
+script.onload = function () {
+    elasticApm.init({
+        serviceName: 'compas-open-scd', // or preferred name, this will be used to filter out results in Kibana
+        serverUrl: 'https://your-own-serverUrl.with.a.specific:port', // replace with serverUrl found in Step 1
+        environment: 'test', // The environment where the service being monitored is deployed, e.g. 'production', 'development', 'test', etc. Default: ''
+    });
+}
+document.querySelector('head').appendChild(script);
+```
+
+### 3. Cloning private git repo using Kubernetes initContainers and Kubernetes Secrets <a name="init-containers"></a>
+Start by generating a personal acccess token in Github and make sure you authorize your token to access your private repo (configure SSO if needed), your token must have checked the write:packages scope checkbox.
+
+Then you must create your Kubernetes Secret:
+```sh
```

Comment:
This can be moved to our internal Repo

Reply:
Would you prefer this particular document to finish at point 2. then?